lib/smart_app_launch/client_suite/access_alcs_interaction_test.rb

```diff
@@ -14,6 +14,13 @@ class SMARTClientAccessAppLaunchConfidentialSymmetricInteraction < Inferno::Test
       using a SMART token obtained using the SMART App Launch EHR launch
       or standalone launch flow.
     )
+    verifies_requirements 'hl7.fhir.uv.smart-app-launch_2.2.0@12',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@23',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@64',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@66',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@229',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@235'
```

Comment:
@235 seems like a better fit for lib/smart_app_launch/client_suite/authorization_request_verification_test.rb. Please check and move/remove if you agree.

@235
[When requesting] an access token via HTTP POST to the FHIR authorization server’s token endpoint URL [the] scope [parameter is] required [and SHALL contain] … the scope of access requested ... following the [SMART Scopes syntax](https://hl7.org/fhir/smart-app-launch/STU2.2/scopes-and-launch-context.html)